exercises/crypto-square/canonical-data.json

```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
       "description": "empty plaintext results in an empty ciphertext",
-      "property": "ciphertext",
+      "property": "cipherText",
```

Comment:
I think ciphertext is a better property than cipherText, so I would revert this change.